config.py

```diff
@@ -152,4 +172,5 @@ class TestingConfig(DevelopmentConfig):
 Z5VVFymXN2n+A6UeWAnuO8/E1inhk99dBzKEGdw=
 -----END RSA PRIVATE KEY-----'''
     SECRET_KEY = 'sekrit!'
-
+    SECURITY_USER_NAME = 'username'
```

Comment:
This is minor, but everywhere uses admin/secret as the default security_user_name and security_user_password.